lib/rex/post/meterpreter/packet_dispatcher.rb

```diff
@@ -88,6 +89,9 @@ def shutdown_passive_dispatcher
     self.send_queue = []
     self.recv_queue = []
     self.waiters    = []
+
+    self.tlv_log_file.close unless self.tlv_log_file.nil?
+    self.tlv_log_file = nil
```

Comment:
Would it make sense to create your own `initialize_tlv_logging` and `shutdown_tlv_logging` ? This logic seems out of place 🤔